website/docs/integrations/airflow/preflight-check-class.md

```diff
@@ -0,0 +1,284 @@
+---
+sidebar_position: 4
+title: Preflight Check Class
+---
+# Preflight Check Class
+
+## Purpose
+
+In some cases, validation of OpenLineage setup in an Airflow instance might be desired outside Airflow or at the task level within a DAG. In these cases, you can use this Python class instead of the [Preflight Check DAG](https://openlineage.io/docs/integrations/airflow/preflight-check-dag), on which this class is based. 
```

Comment:
I think the issue is without Airflow context the check does not make sense for Airflow, since Airflow integration can be configured from Airflow config, which is not accessible without Airflow.

Reply:
I think you're right, however it's pretty unusual use case - I did not even think about it first.